### DISCUSSION TITLE
Server Folders?

### DISCUSSION THREAD

**Discussion by KenWuqianghao** [cryptoAlgorithm/Swiftcord#40] [Ideas]
Would it be possible to resemble the folder structure for discord servers? Once I login, my previous folder structures were removed and returned to individual servers on the side bar.

**Comment by cryptoAlgorithm** [comment]
There is the ability to handle server folders in DiscordKit, but it isn't implemented in Swiftcord yet. However, Swiftcord doesn't modify any of your existing server folder settings, and they will still be present in the official client. You can join the Discord server to see my findings on guild folders.
